app/webserver/helpers.go

```diff
@@ -149,14 +149,18 @@ func sendDeviceList() {
 	}
 	broadcast <- *message
 }
-func createChat(tel string) *store.Session {
-	return store.SessionsModel.GetByE164(tel)
+func createChat(uuid string) *store.Session {
+	session, err := store.SessionsModel.GetByUUID(uuid)
+	if err != nil {
```

Comment:
We don't check here that the `err` is that the Session is non existent? Is that the only error which can be raised?

Reply:
The second error is the check for an invalid UUID, I've added the check now, too.